src/index.js

```diff
@@ -1,13 +1,39 @@
-export function baseUrl(base) {
+export function baseUrl(baseUrl) {
   // extension code here
 
+  const reAbsUrl = /^[\w+]+:\/\//;
+  baseUrl = baseUrl.trim().replaceAll(/[\/\.]+$/g, '') + '/'; // make sure baseUrl ends with one '/'
```

Comment:
I don't think we want to relax the input by automatically putting a slash at the end of the baseUrl.

The base url doesn't need to be a folder, it could be the current url that has a file name at the end.

For example, if the current url is `https://example.com/path/index.html` and someone uses `marked.use(baseUrl(location.href));` than a relative url will **correctly** remove the `index.html`.

With this PR the user would have to remove `index.html` from the href before using it as a base url.

Reply:
The input is a URL after all not a file path.

Reply:
@UziTech,

Thanks for the input. You got the point. Update is coming in a min.

Cheers,